style.css

```diff
@@ -372,3 +383,84 @@ textarea:focus {
   border-radius: 10px;
   background: #0d151e;
 }
+
+@media (min-width: 768px) {
+  header {
+    height: 100px;
+    padding: 20px;
+  }
+
+  main {
+    height: 650px;
+    background-image: url(images/desktop/Header-bg.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  main > .section-1 {
+    display: flex;
+    flex-direction: column;
+
+    /* align-items: center; */
+    justify-content: center;
+    margin: 150px;
+  }
+
+  .container > nav {
+    display: inline-block;
+    font-weight: 500px;
+    color: #344563;
+  }
+
+  .works > .job {
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
```

Comment:
- [ ] consider matching your design like the one in the Figma template by specifying the correct padding and margins for work cards section 

- [ ] Also consider adding the correct background color for the work section like the Figma design.
- [ ] Kindly add appropriate padding and margin for about section

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/177556111-c46be900-ad43-4ccd-839b-b9b2dd73108f.png)| ![image](https://user-images.githubusercontent.com/53402600/177556259-7e3aaf21-03d6-4e3b-9d76-25aabcd37b8d.png) |
| ![image](https://user-images.githubusercontent.com/53402600/177557163-67b37592-c3cf-476d-8457-03272e198e3d.png) | ![image](https://user-images.githubusercontent.com/53402600/177557238-afb74457-25f1-4087-bd77-fc52f3d432a8.png)
 |